components/butterfly/src/rumor/election.rs

```diff
@@ -303,15 +299,15 @@ mod tests {
     fn create_election(member_id: &str, suitability: u64) -> Election {
         Election::new(
             member_id,
-            ServiceGroup::new(None, "tdep", "prod", None).unwrap(),
+            &ServiceGroup::new(None, "tdep", "prod", None).unwrap(),
```

Comment:
Would this be preferable?
```suggestion
            "tdep.prod",
```